internal/pkg/manifest/rd_web_svc.go

```diff
@@ -186,6 +191,9 @@ func (s RequestDrivenWebService) applyEnv(envName string) (workloadManifest, err
 
 func (s *RequestDrivenWebService) requiredEnvironmentFeatures() []string {
```

Comment:
This has nothing to do with your PR but it's pretty strange to me that this method: https://github.com/aws/copilot-cli/blob/5781c535bc78e5e3a544b855d930256562e52b63/internal/pkg/manifest/loader.go#L53

belongs to `*DynamicWorkloadManifest` 🤔 it doesn't seem relevant to a "dynamic" aspect. 
I'd have expected it to be part of the individual Manifest types.